src/pages/player-profile.tsx

```diff
@@ -20,16 +20,14 @@ const PlayerProfile = () => {
 
   if (user.isLoading) {
     return (
-      <Stack sx={styles.mainContainer}>
+      <Stack width="100%" pt="40vh" alignItems="center">
         <CircularProgress />
       </Stack>
```

Comment:
Can we make it a LoadingPage component so we re-use the same for other pages as well?

Reply:
yeah, we might as well, adding